sherpa/utils/__init__.py

```diff
@@ -1323,6 +1322,10 @@ def print_fields(names, vals, converters=None):
                       numpy.uint8: 'UInt8',
                       numpy.void: 'Void0'
                       }
+        if platform.processor() != 'arm' or platform.machine() != 'arm64':
```

Comment:
I think it would be better to just check whether the attributes exist rather than base it on the platform type (e.g. windows may not have these symbols, and whilst we currently don't support a windows build we should try to avoid hard-coding logic when we can):

```python
try:
    converters[numpy.complex256] = 'Complex256'
except AttributeError:
    pass

try:
    converters[numpy.float128] = 'Float128'
except AttributeError:
    pass
```

Note above code is untested...

Reply:
made the changes you requested

Reply:
Ta